lib/Core/ErrorState.cpp

```diff
@@ -357,7 +372,7 @@ void ErrorState::executeStoreSimple(llvm::Instruction *inst, ref<Expr> address,
 }
 
 ref<Expr> ErrorState::retrieveStoredError(ref<Expr> address) const {
-  ref<Expr> error = ConstantExpr::create(0, Expr::Int8);
+  ref<Expr> error;
```

Comment:
I think here the old code where error is assigned to 0 is safer, because ref<Expr> without content, like the new erorr here may cause segmentation faults. I think the assignment should not affect the output.